src/pages/tutorials/build-an-nft-marketplace/index.md

```diff
@@ -312,24 +344,50 @@ We have finished the smart contract implementation for this first training, let'
 Edit the storage file `nft.storageList.jsligo` as it. (:warning: you can change the `administrator` address to your own address or keep `alice`)
```

Comment:
- Edit "as it"?
- Clarify the warning; it's important because this address is the admin for the NFT collection and the only account that can create NFTs.
- This `:warning:` annotation doesn't seem to do anything